project_closing/README.rst

```diff
@@ -1,21 +1,36 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
-Project closing
 ===============
+Project Closing
+===============
+
+This module extends the functionality of projects to allow automatic
+account analytic archiving when related project is archived, and if
+a project is dearchived, the related analytic account is dearchived as well.
+
+Usage
+=====
 
-Automatic account analytic closing when related project is closed,
-and if a project is opened, the related analytic account will be re-opened.
+To use this module, you need to:
 
+* Go to *Project > Project > Projects*.
+* Edit one.
+* Click on *Archive*.
+* The associated analytic account will also be archived.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
 
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/project-service/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/project-service/issues/new?body=module:%20project_closing%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
-
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/project/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
 
 Credits
 =======
```

Comment:
Add Images section after credits section in order to add oca icon as is in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst file